docs/AWS_doc.md

```diff
@@ -0,0 +1,197 @@
+
+
+
+### Overview
+
+This guide will walk you through the process of deploying and managing applications in an EKS cluster using the [AWS Marketplace](https://aws.amazon.com/marketplace) and [Helm](https://helm.sh/). This guide will show you the steps to deploy the [dNation Kubernetes Monitoring Helm chart](https://github.com/dNationCloud/kubernetes-monitoring) on your EKS cluster with Helm.
+
+Here are the steps you’ll follow in this tutorial:
+
+- Subscribe to the dNation Kubernetes Monitoring using the AWS Marketplace
+- Deploy the dNation Kubernetes Monitoring Helm chart on EKS through Helm
+- Log in and start using dNation Kubernetes Monitoring
+
+The next sections will walk you through these steps in detail.
+
+
+
+### Assumptions And Prerequisites
+
+This guide assumes that:
+
+- You have an active AWS account. If you don’t have this, [create a new account](https://aws.amazon.com/).
+- You have a running EKS cluster with Helm 3.x and *kubectl* installed. 
+
+
+
+### Step 1: Subscribe To The dNation Kubernetes Monitoring Using The AWS Marketplace
+
+At the end of this step, you will have subscribed to the dNation Kubernetes Monitoring  solution in the AWS Marketplace.
+
+Follow these steps:
+
+- Log in to the [AWS Marketplace](https://aws.amazon.com/marketplace).
+
+![log_in](images/aws_log_in.png)
+
+- Search for the dNation Kubernetes Monitoring by entering the search term “dNation kubernetes monitoring” in the search bar at the top.
+
+  
+
+- Select the dNation Kubernetes Monitoring in the list of search results.
+
+  
+
+- On the product detail page, review the details of the solution and click the “Continue to subscribe” button.
+
+![continue_to_subscribe](images/continue_to_subscribe.png)
+
+
+
+- On the product subscription page, select “Continue to configuration” as accept the terms and pricing.
+
+  ![continue_to_configuration](images/continue_to_conf.png)
+
+  
+
+- On the product configuration page, select delivery method as “Container” , select Software version and click the "Continue to Launch".
+
+  ![configuration_page](images/conf_page.png)
+
+  
+
+- On the launch page, you can review your configuration and see the instruction to the given software.
+
+  ![launch_page](images/launch.png)
+
+
+
+### Step 2: Deploy dNation Kubernetes Monitoring Helm Chart On EKS
+
+At the end of this step, you will have deployed dNation Kubernetes Monitoring  on your EKS cluster.
+
+The next step is to deploy dNation Kubernetes Monitoring on your EKS cluster. The easiest way to do this is with a Helm chart.
+
+
+
+Follow these steps:
+
+- Add dNation helm repository and update it.
+
+  ``` bash
+  # Add dNation helm repository
+  helm repo add dnationcloud https://dnationcloud.github.io/helm-hub/
+  helm repo update
+  ```
+
+  
+
+- It is a good practice to install new packages in a separate namespace, as it is easier to manage it this way. Create a new namespace, for example called it "monitoring"
```

Comment:
....for example call it "monitoring"...
OR
...for example called monitoring...